packages/backend/src/graphql/project/mutations/projectUpdateMutationField.ts

```diff
@@ -11,14 +11,15 @@ builder.mutationField('projectUpdate', (t) =>
       data: t.arg({ type: ProjectInput }),
     },
     authScopes: async (_source, { id }) => ({ isAdminByProject: id.toString() }),
```

Comment:
When updating the organization, we should check that the user is org member of the organization.